plugins/plugin-kubeui-client/index.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import './web/css/static/carbon-overrides.css'
+import './web/css/static/kui-tables.css'
+import './web/css/static/kui-ui.css'
+
+require('@kui-shell/plugin-client-common')
```

Comment:
i think we can just extend public-client-default here?

Reply:
solved, thanks!